core/unix/inc/TUnixSystem.h

```diff
@@ -31,11 +31,22 @@
 #include "TTimer.h"
 #endif
 
+#include <thread>
+#include <memory>
+
+namespace std {
+  class thread;
```

Comment:
Thanks Philippe - Zhe, can you run through this guide and try to get the code to compliance?

@pcanal - when contributing to CVMFS, they provide an automated tool (based on cpplint).  I see you reference astyle for managing whitespace.  Do you have anything similarly automated you use?